src/AppBundle/Controller/DefaultController.php

```diff
@@ -14,8 +14,8 @@ class DefaultController extends Controller
     public function indexAction(Request $request)
     {
         // replace this example code with whatever you need
-        return $this->render('default/index.html.twig', [
-            'base_dir' => realpath($this->getParameter('kernel.root_dir').'/..'),
-        ]);
+        return $this->render('default/index.html.twig', array(
```

Comment:
The "Coding Standard" check from fabbot complains when a PR is submitted with the new syntax. Should I re-submit with the new syntax anyway?